lib/shared.js

```diff
@@ -204,7 +204,7 @@ module.exports = {
   },
   overrides: [
     {
-      files: ['*.ts', '*.tsx'],
+      files: ['**/*.ts', '**/*.tsx'],
```

Comment:
This change was required for the tests to work with the new exported flat configs. Not entirely sure why...

Without the change, the test files get ignored in the flat tests.